### PR TITLE
chore: change test report name

### PR DESCRIPTION
**Pull request by evgeniy-ac**

### CODE REVIEW

File: build/Program.cs [hunk 1]
```diff
@@ -311,27 +304,20 @@ await Cli
...
+                "--logger \"console;verbosity=detailed\" " +
+                "--logger \"trx;LogFileName=Results.trx\" " +
                 (IsGitHubActions() ? "--logger GitHubActions " : "") +
                 $"-c {configuration} "
                 )
                 .ToConsole()
                 .ExecuteBufferedAsync(cancellationToken).Task.ConfigureAwait(false);
-
-            MoveAttachmentsToResultsDirectory(resultsDirectory, cmd.StandardOutput);
-            TryRemoveTestsOutputDirectories(resultsDirectory);
```
Comment (FrolYo) [inline build/Program.cs:334]: maybe remove TryRemoveTestsOutputDirectories?

File: docker-compose.unit-tests.yml [hunk 2]
```diff
@@ -7,11 +7,12 @@ services:
       args:
         GITHUB_TOKEN: ${GITHUB_TOKEN}
     volumes:
-      - ./artifacts/tests/docker-output/:/src/artifacts/tests/output/
+      - ./output:/output
```
Comment (FrolYo) [inline docker-compose.unit-tests.yml:10]: why not artifacts? we do everything via artifacts

File: docker-compose.unit-tests.yml [hunk 3]
```diff
@@ -7,11 +7,12 @@ services:
...
     environment:
       NPM_READ_TOKEN: ${NPM_READ_TOKEN}
     command: >-
       /bin/sh -c "
       dotnet build --nologo --no-restore --configuration Debug &&
-      dotnet run --project build --configuration Release --no-build --no-launch-profile -- --configuration Debug unit-tests
+      dotnet run --project build --configuration Release --no-build --no-launch-profile -- --configuration Debug unit-tests &&
+      cp -R /src/tests/. /output
```
Comment (FrolYo) [inline docker-compose.unit-tests.yml:17]: why do we copy test sources to output?

**Review by FrolYo — CHANGES_REQUESTED** [review]

File: docker-compose.unit-tests.yml (hunk 3 above)
Comment (evgeniy-ac) [reply docker-compose.unit-tests.yml:17]: we copy all to be able to use wildcard in a github action 
```
path: './output/**/*.trx
```

File: docker-compose.unit-tests.yml (hunk 2 above)
Comment (evgeniy-ac) [reply docker-compose.unit-tests.yml:10]: I can try but probably we already use the artifacts folder to build tests. I don't know why

File: docker-compose.unit-tests.yml (hunk 3 above)
Comment (FrolYo) [reply docker-compose.unit-tests.yml]: aren't they saved to artifacts folder?

Comment (evgeniy-ac) [reply docker-compose.unit-tests.yml]: saved where?

Comment (evgeniy-ac) [reply docker-compose.unit-tests.yml]: where?

File: docker-compose.unit-tests.yml (hunk 2 above)
Comment (evgeniy-ac) [reply docker-compose.unit-tests.yml]: done

File: docker-compose.unit-tests.yml (hunk 3 above)
Comment (alexyakunin) [reply docker-compose.unit-tests.yml]: @evgeniy-ac @FrolYo did you figure out what has to be done here?

**Review by alexis-kochetov — APPROVED** [review]

**Pull request merged**